docs/medical-api/api-reference/document/list-documents.mdx

```diff
@@ -7,11 +7,6 @@ api: "GET /medical/v1/document"
 This endpoint returns the document references available at Metriport which are associated with the
 given Patient.
 
-It also returns the status of querying document references across HIEs, indicating whether
-there is an asynchronous query in progress (status `processing`) or not (status `completed`).
-
-If the query is in progress, you will also receive the total number of documents to be queried as well as the ones that have already been completed.
-
```

Comment:
Yeah - just looking at this endpoint in isolation, it doesn't seem obvious as to why there'd be a status in the response.

Wondering if we should go as far as to remove the status from this response entirely - now that we have the new endpoint?

Reply:
I think Dima's point makes sense. From a Cx's perspective i envision the flow to be

- Create a patient 
- Either list docs and if empty query or just go straight to querying.
- Listen via webhooks or this new endpoint if completed
- List docs

Reply:
We can defs remove the status from the endpoint, just thinking about impact to cxs... do you know if they're using it? If so we just need to communicate beforehand and give some time for them to update it

Reply:
Should I just remove the status from the `GET /document` endpoint then?

Reply:
@Goncharo @Orta21 confirm we want to remove the status from the /documents endpoint?

Reply:
@leite08 if we all agree this will be better design - we can remove it, however we may need to consider how the dashboard works here as well.

If it's a bunch more work, perhaps we can just ticket the removal, and at least explain in the docs that the status will be returned, and to learn more see the `Get Document Query status` page?

Reply:
Updated according to chat on https://metriport.slack.com/archives/C04CXLSAF7V/p1687442602028909